scenarios/intersections/4lane_t/requirements.txt

```diff
@@ -1,4 +1,4 @@
 --extra-index-url http://localhost:8080
 
-open-agent==0.1.189
+open-agent==0.0.0
 rl-agent==0.1.1
```

Comment:
This rl-agent does not work as intended at the moment either.

Reply:
cool, I removed this from requirements